.github/workflows/build.yml

```diff
@@ -12,7 +12,6 @@ on:
     branches:
       - master
   pull_request:
-    branches: '*'
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏was this not necessary?

Reply:
No, and it seems like since it’s following glob behavior the workflows don’t run when your PR targets a branch with a `/` in the name. (The workflow from GitHub that I based the size comparison off of doesn’t specify `branches`)